pkg/rules/remotewrite/series.go

```diff
@@ -0,0 +1,256 @@
+package remotewrite
+
```

Comment:
Let's comment this file was copied! 

Also Let's add TODO to use Prometheus package when avaialble https://github.com/prometheus/prometheus/pull/8785